plugins/system/cpu.go

```diff
@@ -38,13 +54,53 @@ func (s *CPUStats) Gather(acc plugins.Accumulator) error {
 		add(acc, "guest", cts.Guest, tags)
 		add(acc, "guestNice", cts.GuestNice, tags)
 		add(acc, "stolen", cts.Stolen, tags)
+		add(acc, "busy", busy, tags)
+
+		// Add in percentage
+		lastCts := s.lastStats[i]
+		lastBusy, lastTotal := busyAndTotalCpuTime(lastCts)
+		busyDelta := busy - lastBusy
+		totalDelta := total - lastTotal
+
+		if totalDelta < 0 {
+			return fmt.Errorf("Error: current total CPU time is less than previous total CPU time")
+		}
+
+		if totalDelta == 0 {
+			return nil
```

Comment:
This should be `continue` instead of `return nil`, otherwise if we can't get percentages for one CPU then we get no stats at all for the rest.